e2e_playwright/st_dataframe_config.py

```diff
@@ -513,3 +513,22 @@
         ],
     )
 )
+
+st.subheader("Hierarchical headers")
+
+df = pd.DataFrame(
+    np.random.randn(3, 5),
+    index=["A", "B", "C"],
+    columns=pd.MultiIndex.from_tuples(
```

Comment:
for my personal understanding (pandas related): the function says `from_tuples`, but you pass a `list[list]` here. Is that correct or does it just accidentally work?

Reply:
> Is that correct or does it just accidentally work?
I think it works with any iterable inside the list. But I changed it to a list of tuples 👍